tasks.py

```diff
@@ -262,6 +263,16 @@ def type_check(
         bin = "mypy"
 
     ge_pkgs = [f"great_expectations.{p}" for p in packages]
+
+    if check_source_files:
+
+        for stub_file in GX_ROOT_DIR.glob("**/*.pyi"):
+            source_file = stub_file.with_name(  # TODO:py3.9 .with_stem()
+                f"{stub_file.name[:-1]}"
+            )
+            relative_path = source_file.relative_to(GX_ROOT_DIR.parent)
+            ge_pkgs.append(str(relative_path))
```

Comment:
With this change, we can ensure the safety of our implementation `.py` files.

https://dev.azure.com/great-expectations/great_expectations/_build/results?buildId=225035&view=logs&j=b160066a-9799-5484-6f26-c8df60db6a40&t=1a7e8769-7782-59c9-09ad-b9f0d1a5bd8b